starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java

```diff
@@ -83,7 +85,9 @@ public Optional<Inventory> update(String id, Inventory inventory) {
    * @return Deleted Inventory.
    */
   public Optional<Inventory> delete(String id) {
```

Comment:
What if the string is empty or null?

Reply:
Added Assert.hasText method to prevent null & empty string arguments